websites/management/commands/markdown_cleaning/rules/link_to_external_resource.py

```diff
@@ -251,11 +251,18 @@ def replace_match(
         config = self.starter_lookup.get_config(starter_id)
         link_text = toks.link.text
 
+        # Unescape markdown characters in the link text for use in shortcode
+        # Markdown may escape backticks and square brackets that should not be
+        # escaped in the shortcode title parameter
+        unescaped_link_text = (
+            link_text.replace(r"\`", "`").replace(r"\[", "[").replace(r"\]", "]")
```

Comment:
It would be better if this were a utility function, defined elsewhere. Otherwise, there is code duplication here and below.

Reply:
Updated in 0999ad92f1b567e07955c4553f3c40e3255c5ec7.